tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -1120,72 +1086,65 @@ TEST_P(DistributedTransformerTest, Backward) {
   fusion->addInput(ln0_b);
   fusion->addInput(ln0_mean);
   fusion->addInput(ln0_rstd);
+  fusion->addInput(mha_linear0);
   fusion->addInput(mha_linear1);
+  fusion->addInput(mlp_linear0);
 
   // Recomputation: Recompute, mha linear0, qkv, mlp linear0, and mlp gelu.
```

Comment:
I believe this needs to be updated, e.g., mha_linear0 and mlp_linear0 will no longer be recomputed.